packages/lesswrong/components/users/UsersMenu.tsx

```diff
@@ -270,7 +270,7 @@ const UsersMenu = ({classes}: {
         : null,
   } as const;
 
-  const hasBookmarks = (currentUser?.bookmarkedPostsMetadata.length ?? 0) >= 1;
+  const hasBookmarks = isEAForum || (currentUser?.bookmarkedPostsMetadata.length ?? 0) >= 1;
```

Comment:
This `(currentUser?.bookmarkedPostsMetadata.length ?? 0) >= 1` appeared to be broken by the recent changes. This may have just been a migration thing, because I'm now struggling to reproduce the broken state. But either way on EAF we in fact always want to show this so people can access the "continue reading" section.

Reply:
Hm... I think there needs to be a follow-up to drop `bookmarkedPostsMetadata` from the database post-migration, and switch it to be a resolver field

Reply:
makes sense regarding making it a resolver field